assets/js/pages/index.js

```diff
@@ -76,8 +76,21 @@ removeErrorMessage() {
           <section className="phx-hero">
             <h1>Welcome to Tic Tac Toe!</h1>
           </section>
-          <section>
-            <Message message={this.state.message} error={this.state.error}/>
+          <section> 
+            { (this.state.gameStatus == "winner" || this.state.gameStatus == "draw") ? 
```

Comment:
I know ternary operators are fun to use but at a certain point, they start being hard to read. If you really want to keep the operator, consider extracting the conditional and the two return values into separate methods.